### PR TITLE
Implement NativeAnimated modulus node on Android

### PR DESCRIPTION
**Pull request by ryangomba**
This diff implements ModulusAnimatedNode on Android, bringing Android up to date with JS and iOS native animation APIs.

### CODE REVIEW

**Comment by ryangomba** [comment]
@janicduplessis I have a few PRs (this included) for your review when you get a chance.

File: ReactAndroid/src/main/java/com/facebook/react/animated/ModulusAnimatedNode.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package com.facebook.react.animated;
```
Comment (janicduplessis) [inline ReactAndroid/src/main/java/com/facebook/react/animated/ModulusAnimatedNode.java]: Add copyrights header

File: ReactAndroid/src/main/java/com/facebook/react/animated/ModulusAnimatedNode.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import com.facebook.react.bridge.JSApplicationCausedNativeException;
+import com.facebook.react.bridge.ReadableArray;
+import com.facebook.react.bridge.ReadableMap;
+
+/*package*/ class ModulusAnimatedNode extends ValueAnimatedNode {
+
+  private final NativeAnimatedNodesManager mNativeAnimatedNodesManager;
+  private final int inputNode;
```
Comment (janicduplessis) [inline ReactAndroid/src/main/java/com/facebook/react/animated/ModulusAnimatedNode.java]: Nit: Add `m` prefix for ivars

File: ReactAndroid/src/main/java/com/facebook/react/animated/ModulusAnimatedNode.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  private final int modulus;
```
Comment (janicduplessis) [inline ReactAndroid/src/main/java/com/facebook/react/animated/ModulusAnimatedNode.java]: same here

**Comment by janicduplessis** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**